compiler/main.nim

```diff
@@ -281,7 +281,7 @@ proc mainCommand*(graph: ModuleGraph) =
     else:
       rawMessage(conf, errGenerated, "'run' command not available; rebuild with -d:tinyc")
   of cmdDoc0: docLikeCmd commandDoc(cache, conf)
-  of cmdDoc2:
+  of cmdDoc2:  # the 'nim doc' command (aka doc2)
```

Comment:
or simply rename cmdDoc2 to cmdDoc, making it self-documenting

Reply:
agree, done.